x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -67,9 +67,7 @@ export function SparkPlot({
       alignItems="flexEnd"
       style={flexGroupStyle}
     >
-      <EuiFlexItem grow={false} style={{ whiteSpace: 'nowrap' }}>
-        {valueLabel}
-      </EuiFlexItem>
+      <EuiFlexItem>{valueLabel}</EuiFlexItem>
```

Comment:
Another option could be shortening big numbers like 1000 > 1k, 1000000 > 1m, and etc

Reply:
Nice idea, my concern is that we are losing precision and for such cases, I think it might be important to know the exact number - also this will work for different labels not only numbers.